blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+
+[See our prototype of Automation in action](https://about.sourcegraph.com/product/automation#see-it-in-action) and let us know if you are interested in learning more, or if Automation would benefit you and your company.
+
+## Beta release of fast and precise LSIF-based code intelligence
+
+TO DO: VIDEO?
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Each Sourcegraph instance ships with basic code intelligence that provides jump to definition and find references for every language. Basic code intelligence works well for many of our customers, but some customers have been asking for more precise code intelligence. We previously used language servers to provide precise code intelligence, but they were complex to configure, required separate deployment, and were slow to initialize.
+
+Our solution to fast and precise code intelligence is with LSIF. We first wrote about LSIF (Language Server Index Format) in [code intelligence using LSIF](https://about.sourcegraph.com/blog/code-intelligence-with-lsif). This blogpost gives an overview of what LSIF is, and why we are working to support it.
+
+In Sourcegraph 3.8, LSIF-backed code intelligence is available for beta testing by early adopters. The currently supported languages (ones with LSIF indexers) are [TypeScript](https://github.com/microsoft/lsif-node), [Go](https://github.com/sourcegraph/lsif-go), [Python](https://github.com/sourcegraph/lsif-py), and [C/C++](https://github.com/sourcegraph/lsif-cpp). Check out the documentation for [details on how to try it out on your code](https://docs.sourcegraph.com/user/code_intelligence/lsif).
+
+## Search result tabs for improved resuslt type discoverability
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Many users were not aware of the different search result types that are supported by Sourcegraph, such as diff or symbol results. We have added tabs to the search results page to surface the different result types and make it quick and easy to navigate between them.
+
+## Out of the box monitoring of your Sourcegraph instance
+
```

Comment:
I think a screenshot or two would suffice unless we demonstrate how to get this working (not sure what's involved here).

@uwedeportivo: Are there docs we can link to for configuration etc.

Reply:
nope

Reply:
Docs and better dashboards are coming with the next release. This is just adding an out of the box access to the monitoring tools.